docs/css/style.css

```
@@ -188,6 +188,14 @@ img.screenshot {
   max-width: 100%;
 }
 
```

Adding center styling to the sponsorship `<aside>`. See https://github.com/mochajs/mocha/pull/3765/files#r262465728

```suggestion
.sponsorship {
    display: flex; 
    justify-content: center;
}

```